appmap/pytest.py

```diff
@@ -120,8 +127,8 @@ def pytest_pyfunc_call(pyfuncitem):
     path = os.path.join(session.appmap_path, fname)
 
     def write_recording(r):
-        with open(path, 'wb') as appmap:
-            appmap.write(generation.dump(r, metadata))
+        with open(path, 'wb') as appmap_file:
```

Comment:
AppMaps are only ever files, so this looks like a stutter.

Reply:
This is to make pylint shut up about naming conflict with `appmap` module.